wiz-admission-controller/templates/_helpers.tpl

```diff
@@ -40,6 +40,16 @@ If release name contains chart name it will be used as a full name.
 {{- end }}
 {{- end }}
 
+{{- define "wiz-hpa-enforcer.name" -}}
+{{- $name := "wiz-hpa-enforcer" }}
+{{- default $name .Values.hpa.enforcerNameOverride | trunc 63 | trimSuffix "-" }}
```

Comment:
This is not in the values
Anyway you can use the deployment name (include "wiz-admission-controller.fullname") and add a suffix "hpa".

Reply:
Done, I changed it to:
```
{{- define "wiz-hpa-enforcer.name" -}}
{{- $name := printf "%s-hpa" (include "wiz-admission-controller.fullname" .) }}
{{- default $name .Values.hpa.enforcerNameOverride | trunc 63 | trimSuffix "-" }}
{{- end }}
```

Reply:
`.Values.hpa.enforcerNameOverride` doesn't exist in the valus.yaml

Reply:
Why so you need `{{- default $name .Values.hpa.enforcerNameOverride | trunc 63 | trimSuffix "-" }}`?

Reply:
I did it for consistency with our other objects and I've added:
```
# Horizontal Pod Autoscaling support.
hpa:
  enabled: false
  enforcerNameOverride: "" # Override the HPA name for the Wiz Admission Controller deployment.
  auditLogsNameOverride: "" # Override the HPA name for the Wiz Audit Log Collector deployment.
```

Let me know if you think that's overkill and we could get rid of it.

Reply:
I think it's overkill 😋

Reply:
Done